src/Jeffijoe.MessageFormat.Tests/Formatting/Formatters/PluralFormatterTests.cs

```diff
@@ -74,6 +74,56 @@ public void ReplaceNumberLiterals(string input, string expected)
             var actual = subject.ReplaceNumberLiterals(new StringBuilder(input), 1337);
             Assert.Equal(expected, actual);
         }
+        
+        [Theory]
+        [InlineData(0, "днів")]
+        [InlineData(101, "день")]
+        [InlineData(102, "дні")]
```

Comment:
Here I tested in the same language the pluralizer is for, it's easier to see whether the rules are correct for a native speaker